### ISSUE TITLE
Build Fails After Last Commit on Master Branch

### ISSUE THREAD

**Issue by benjschro** [SFML/CSFML#147]
Simply put, the HEAD of master (7627ef1c5de097ee56f484979941611579d05c72) does not build.  I was able to go back a single commit (cdc9bc6a2197cb2670c25673ba64c76c2a254c97) and it builds just fine.

**Comment by eXpl0it3r** [comment]
[CI](https://ci.appveyor.com/project/eXpl0it3r/csfml/builds/26049941) had no issue building CSFML, so what errors are you running into?

**Comment by benjschro** [comment]
I've tried on both Windows and a Linux VM - same error:
```
/root/CSFML/src/SFML/Graphics/Texture.cpp:186:1: error: expected unqualified-id before '{' token
  186 | {
```
If you look, there is a trailing semicolon after the function definition.  If I remove it, it then gives the following error:
```
/root/CSFML/src/SFML/Graphics/Texture.cpp:189:59: error: no matching function for call to 'sf::Texture::update(sf::Texture* const&, unsigned int&, unsigned int&)'
  189 |     CSFML_CALL_PTR(destination, update(texture->This, x, y));
```

**Comment by eXpl0it3r** [comment]
Ah there's a semicolon that shouldn't be there. Interesting that it builds fine in the CI, maybe it didn't grab the latest source. 🤔 

Either way, there's already a PR for this: #145

**Comment by eXpl0it3r** [comment]
Fixed with #143
